src/common/proxy_wasm/ngx_proxy_wasm_properties.c

```diff
@@ -599,6 +625,12 @@ static pwm2ngx_mapping_t  pw2ngx[] = {
       ngx_null_string,
       &not_supported, NULL },
 
+    /* host properties */
```

Comment:
Oh, I forgot that custom FFI properties are referred to as host properties. 

Yeah, it's surprising that `worker_id` isn't a Nginx variable. That seems to be [intentional](https://forum.nginx.org/read.php?29,250065,250079#msg-250079), though.

Reply:
@thibaultcha, after a conversation with @casimiro I suggested that, since Nginx has reasons for not wanting to expose worker_id and Caio only needs it for running tests, then an idea would be to set this in debug mode only (perhaps having ngx_wasm_debug_module call the properties API to set a debug-only host property).

This would keep things cleaner API-wise and lower the impact of the naming choice. A name would still need to be picked though; since I think Nginx variables don't have dots in them, perhaps the `ngx.wasmx.*` is available without clashing? (Maybe even `ngx.wasmx.debug.*` if we want to make clear this is for debug/testing purposes only)

Reply:
@casimiro Are you still planning on making this change?

Reply:
Yes, I am.

Reply:
Actually, I learned that `ngx_proxy_wasm_properties_set_ffi_handlers` can only be called from within the context of a request. However, the test-case that requires access to `worker_id` does so in `on_tick`, outside the context of a request. 

I then decided to just enclose the function `get_worker_id` and its inclusion in the properties list between `NGX_DEBUG` ifdefs and remove the test-case that used to document the retrieval of the `worker_id` property.

Reply:
Sounds good!